ext/event/backend/epoll.c

```diff
@@ -236,6 +238,73 @@ VALUE Event_Backend_EPoll_select(VALUE self, VALUE duration) {
 	return INT2NUM(count);
 }
 
+VALUE rb_process_status_new(rb_pid_t pid, int status, int error) {
```

Comment:
Please rename it `Event_Backend_process_status_new` and move to `backend.c` (which doesn't exist).

Reply:
`rb_process_status_new` is an internal method inside the `process.c`. I'm checking how to include it properly.